nss-day-cohort-39/tabloid-full-stack-coulees
# Subscribe to Authors

### Changes
- Added "Subscribe" button to post details page for posts that the current user did not write
- Added necessary functions in provider/API to add subscription
- A user can view the posts of authors they are subscribed to on the home page
- Prevented users from subscribing to the same user twice
- Added unsubscribe feature to post details page

### Steps to test
1. Check that the "Subscribe" button appears on posts that you have not written and does not appear on posts that you have written
2. Subscribe to an author
3. Ensure that that author's published posts appear on your home page
4. Check that when you view a post from an author you are subscribed to, you see a "Subscribed" badge and not a button
5. Check that unsubscribe feature works on post details page
6. Be sure to check the edit and delete features for posts that you have written

_I expect some problems with this PR because I had a lot of trouble with this branch_
